src/style.css

```diff
@@ -0,0 +1,86 @@
+body {
+  margin: 0;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  height: 300px;
+  width: 300px;
+  flex-basis: 100%;
+}
+
+.block--1 {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.block--2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.block--3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.block--4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.block--5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.block--6 {
+  background-color: rgb(0%, 0%, 0%);
+}
+
+@media (min-width:600px) {
+  .block {
+    flex-basis: 50%;
+  }
+
+  .block--3 {
+    order: 2;
+  }
+
+  .block--4 {
+    order: 1;
+  }
+
+  .block--5 {
+    order: 3;
+  }
+
+  .block--6 {
+    order: 4;
+  }
+}
+
+@media (min-width:900px) {
+  .block {
+    flex-basis: 33.3%;
```

Comment:
```suggestion
    flex-basis: 33.33%;
```
example
33.3%
<img width="628" alt="Screenshot 2023-01-19 at 11 07 45 AM" src="https://user-images.githubusercontent.com/60295799/213400920-515b3241-f6ce-42c8-925f-c43ebb77d339.png">

33.33%
<img width="628" alt="Screenshot 2023-01-19 at 11 08 08 AM" src="https://user-images.githubusercontent.com/60295799/213401021-9d71991f-9bd5-4db9-b7dd-a988cab76707.png">